masschallenge/django-accelerator
# [AC-9545] 0. Add new fields to the Partner Model

Changes made to [AC-9545](https://masschallenge.atlassian.net/browse/AC-9545):
Added new fields to the Partner Model

How to test:
- Check out to this branch.
- Ensure that the definition of done for [AC-9545](https://masschallenge.atlassian.net/browse/AC-9545) is met.

[Sibling PR](url)
